bin/cqlsh.py

```diff
@@ -2188,6 +2205,28 @@ def read_options(cmdlineargs, environment):
     options.password = maybe_ensure_text(options.password)
     options.keyspace = maybe_ensure_text(options.keyspace)
 
+    serial_levels = [cassandra.ConsistencyLevel.SERIAL, cassandra.ConsistencyLevel.LOCAL_SERIAL]
+
+    try:
+        cl = cassandra.ConsistencyLevel.name_to_value[options.consistency_level.upper()]
+        if cl in serial_levels:
+            raise KeyError
+    except KeyError:
+        cl = cassandra.ConsistencyLevel.ONE
+        parser.error('"{}" is not a valid consistency level, falling back to {}'
```

Comment:
We either need to change the msg here or change the behavior. When I ran 

```
$ bin/cqlsh --consistency-level X
Usage: cqlsh.py [options] [host [port]]

cqlsh.py: error: "X" is not a valid consistency level, falling back to ONE
~/Src/cassandra:[STAR-432-ds-trunk ?]$ echo $?
2
```
cqlsh exited (no fallback, just exit).

Reply:
Here is the source of the problems: DSP-20918
https://github.com/riptano/bdp/commit/138cfb1f0f23b9f970fd9b6f74da7825ff1e2057

But I am going to rather fail badly.

Reply:
Should work as expected: fail with proper msgs not suggesting any fallback.